resources/core/charts/console-backend-service/templates/policy.yaml

```diff
@@ -0,0 +1,13 @@
+{{ if .Values.global.isLocalEnv }}
+apiVersion: authentication.istio.io/v1alpha1
```

Comment:
can we also add here a business comment on why this is needed at all? 

Right now is a magic settings without any background :/